Tests/scripts/hook_validations/docker.py

```diff
@@ -225,8 +232,8 @@ def parse_docker_image(docker_image):
         """
         if docker_image:
             try:
-                tag = re.findall(r'(demisto\/.+):.+', docker_image, re.IGNORECASE)[0]
-                name = re.findall(r'demisto\/.+:(.+)', docker_image, re.IGNORECASE)[0]
+                name = re.findall(r'(demisto\/.+):.+', docker_image, re.IGNORECASE)[0]
+                tag = re.findall(r'demisto\/.+:(.+)', docker_image, re.IGNORECASE)[0]
```

Comment:
why is that separated to two why not `name, tag = re.findall(r'(demisto\/.+):(.+)', docker_image, re.IGNORECASE)[0]`

Reply:
fixed